### PR TITLE
Entry/Profile Form Updates

### LINKED ISSUE #1059: Update style of pulse entry forms
Need to update the Pulse page forms for the 'profile' and 'entry' pages:

Desktop:
https://redpen.io/ede8cba6116ad234e2
https://redpen.io/xhd5029bc3a098fdcc

Mobile:
https://redpen.io/cmd2fe6c2b0d8b4b3e
https://redpen.io/uvd52362e1ecc9003a

cc @youriwims @mmmavis Let me know if you need any more guidance for these. All files are in Sketch.

@gideonthomas @xmatthewx

### PR DESCRIPTION
**Pull request by youriwims**
Closes #1059

https://network-pulse-staging-pr-1107.herokuapp.com/add
https://network-pulse-staging-pr-1107.herokuapp.com/myprofile

### CODE REVIEW

**Comment by natalieworth** [comment]
Looks great @youriwims Only thing that I noticed was in Firefox there is this blue thing that pops up when I type my name on the profile page:
Screen Shot 2019-07-11 at 9 22 49 AM

**Comment by youriwims** [comment]
@natalieworth can you try disabling any plugins you have installed on Firefox?

**Comment by natalieworth** [comment]
> @natalieworth can you try disabling any plugins you have installed on Firefox? Also, did you access this on Android mobile or laptop?...if you did access it on an Android device, that's just a normal android feature 🤖

I tried it on my laptop. It could be an add on I have on though. I have so many. I just tried it and it doesn't seem to be appearing anymore, so all good :D

File: scss/form.scss [hunk 1]
```diff
@@ -73,9 +75,21 @@ span.char-limit {
   }
 }
 
+.form-label {
```
Comment (alanmoo) [inline scss/form.scss]: I don't see the font-size and line-height doing anything when I disable them in the inspector, and then it seems the margin gets overridden by the following selector. Can you double-check what this is all doing and make sure it's necessary? It feels like multiple ways to attack the same core elements.
image

File: pages/add/add.scss [hunk 2]
```diff
@@ -51,15 +59,53 @@
...
+      }
+    }
+  }
+
+  fieldset.issues {
+    margin-bottom: 36px;
+  }
+
+  fieldset.help_types {
+    margin-bottom: 48px;
```
[the remark this answers is not in the document]
Comment (youriwims) [reply pages/add/add.scss]: Is it best to stick with `px`?

File: pages/add/add.scss [hunk 3]
```diff
@@ -51,15 +59,53 @@
...
+    textarea {
+      height: 4.625rem;
+    }
+  }
+
+  p.project-image {
+    &::after {
+      @include form-helper-text("(only submit images that you have permission to use in this context)");
+
+      margin-left: 0;
```
[the remark this answers is not in the document]
Comment (youriwims) [reply pages/add/add.scss]: Thank you for pointing this out.

**Comment by mmmavis** [comment]
image


@youriwims can you update this PR so it's against `master`?

File: pages/profile-edit/form/fields.js [hunk 4]
```diff
@@ -33,58 +30,53 @@ module.exports = function(showLongBio) {
...
+      labelClassname: `h5-heading bio`,
       fieldClassname: `form-control`,
       validator: [validator.maxLengthValidator(140)],
       charLimit: 140,
       charLimitText: function(charCount, charLimit) {
         return charLimit - charCount;
       }
     },
-    twitter: {
+    Twitter: {
```
Comment (mmmavis) [inline pages/profile-edit/form/fields.js]: Why are we changing this? This changes the input field's `name` attribute value from `twitter` to `Twitter`.

File: pages/profile-edit/form/fields.js [hunk 5]
```diff
@@ -33,58 +30,53 @@ module.exports = function(showLongBio) {
...
       type: `text`,
       label: `Twitter`,
       placeholder: `https://twitter.com/username`,
+      labelClassname: `h5-heading`,
       fieldClassname: `form-control`,
       validator: [validator.urlValidator(`Twitter`)]
     },
-    linkedin: {
+    Linkedin: {
```
Comment (mmmavis) [inline pages/profile-edit/form/fields.js]: Same q: why are we changing this?

File: pages/profile-edit/form/fields.js [hunk 6]
```diff
@@ -33,58 +30,53 @@ module.exports = function(showLongBio) {
...
       type: `text`,
-      label: `LinkedIn`,
+      label: `Linkedin`,
       placeholder: `https://linkedin.com/in/username`,
+      labelClassname: `h5-heading`,
       fieldClassname: `form-control`,
       validator: [validator.urlValidator(`LinkedIn`)]
     },
-    github: {
+    Github: {
```
Comment (mmmavis) [inline pages/profile-edit/form/fields.js]: Same q: why are we changing this?

File: pages/profile-edit/form/fields.js (hunk 4 above)
Comment (youriwims) [reply pages/profile-edit/form/fields.js]: It also corresponds to the element's class and in this case, adds a glyph before each `label`.

**Comment by youriwims** [comment]
> I will review this PR more thoroughly once it's been up to date and set to compared against `master`.

@mmmavis see #1115.

**Pull request closed without merging**